### DISCUSSION TITLE
Type conversion when using a native AOT library on platform specific application

### DISCUSSION THREAD

**Discussion by Take-A-Byte** [dotnet/runtime#100387] [Q&A] [answered]
Is there documentation for type conversion when a library is compiled into native AOT library and used in a platform-dependent project (e.g. .so is used in Kotlin Android)?

**Comment by jkotas** [comment, marked as answer]
Does https://learn.microsoft.com/en-us/dotnet/standard/native-interop/type-marshalling#default-rules-for-marshalling-common-types help?
